### ISSUE TITLE
/esm5/ isn't an ES module

### ISSUE THREAD

**Issue by appsforartists** [aleris/zxing-typescript#23]
I expected this to use the ES2015 `export { BrowserQRCodeSvgWriter }` syntax.  Instead, it uses CommonJS's `require`.

I suspect your tsconfig needs to be updated to export proper ES modules (`module: "ES2015"`), or your transpiled folder should be renamed `cjs`.

https://unpkg.com/@zxing/library@0.14.2/esm5/browser/BrowserQRCodeSvgWriter.js

**Comment by odahcam** [comment]
Yes, it's commonJS, an ES6 build output shall be added to solve this.
